pkgs/top-level/all-packages.nix

```diff
@@ -29105,6 +29105,8 @@ with pkgs;
 
   teensy-udev-rules = callPackage ../os-specific/linux/teensy-udev-rules { };
 
+  sensible-side-buttons-bin = callPackage ../os-specific/darwin/sensible-side-buttons-bin { };
```

Comment:
```suggestion
  sensible-side-buttons = callPackage ../os-specific/darwin/sensible-side-buttons { };
```

Reply:
I was recommended to use `bin` in another one of my pull requests. Without much information I guessed that when the package is not built from source we can package it as `bin`.

Since the binary itself will be `sensible-side-buttons-bin`, you requested this change?

Thank you.

Reply:
Let me ask to one of my colleague about this... 

@AndersonTorres What's your point of view on this?

Reply:
@drupol How do you want to proceed here? 🙂